tutorials/navigation/navigation_debug_tools.rst

```diff
@@ -59,3 +59,31 @@ The connections can be made visible through geometry with the navigation debug `
 .. note::
 
     Edge connections are only visible when the NavigationServer is active.
+
+Debug Performance
+-----------------
+
+To measure NavigationServer performance a dedicated monitor exists that can be found within the Editor Debugger under `Debugger->Monitors->NavigationProcess`.
+
+.. image:: img/navigation_debug_performance1.webp
+
+NavigationProcess shows how long the NavigationServer spend updating its internals this update frame in milliseconds.
+NavigationProcess works similar to Process for visual frame rendering and PhysicsProcess for collision and fixed updates.
+
+NavigationProcess accounts for all updates to ``navigation maps``, ``navigation regions`` and ``navigation agents`` as well as the entire ``avoidance calculations`` for the update frame.
```

Comment:
Is ``avoidance calculations`` in this sentence referring to a specific thing, like a property named ``avoidance calculations``? or does this just mean every avoidance calculation? If this means every avoidance calculation and not a singular thing "the entire" should be replaced with "all the" in this sentence,

Reply:
means every avoidance calculation